whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -5117,14 +5117,38 @@
               ]
             }
           }
+        },
+        {
+          "name": "ingore $b",
+          "source": {
+            "010": {
+              "ind1": " ", "ind2": " ",
+              "subfields": [{"a": "a 54009564"}, {"b": "IGNORED"}]
+            }
+          },
+          "result": {
```

Comment:
Excellent! Also just needs a "normalized" for revert (for both specs) without the ignored subfield.

Reply:
Good call, will do!